### PR TITLE
Add github action to call oci-image-builder

### PR DESCRIPTION
**Pull request by KacperMalachowski**
**Description**

Add basic gh action to trigger oci-image-builder, it's first PR related directly to GH action

Changes proposed in this pull request:

- Update reusable workflow to utilize newly added github aciton
- Add action to trigger oci-image builder using image-builder docker container

**Related issue(s)**

See #9978

### CODE REVIEW

**Comment by KacperMalachowski** [comment]
@dekiel it doesn't receive a repository or ref as it's read inside the docker image directly. You probably mismatch reusable workflow inputs with actions inputs.

Also all necessary flags are set for the image-builder to run as a GitHub action. Please verify it with code as documentation is not updated yet.

File: .github/workflows/pull-image-builder-test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      - ".github/workflows/image-builder.yml"
+      - ".github/actions/expose-jwt-action/**"
+      - ".github/actions/image-builder/**"
+
+jobs:
+  test_image_builder:
+    uses: ./.github/workflows/image-builder.yml # Usage: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+    with:
+      name: test-infra/ginkgo
+      dockerfile: prow/images/ginkgo/Dockerfile
```
Comment (Sawthis) [inline .github/workflows/pull-image-builder-test.yml:17]: does it work without providing name of the env file? 
https://github.com/kyma-project/test-infra/blob/main/prow/images/ginkgo/envs

**Review by Sawthis — CHANGES_REQUESTED** [review]

File: .github/workflows/pull-image-builder-test.yml (hunk 1 above)
Comment (KacperMalachowski) [reply .github/workflows/pull-image-builder-test.yml:17]: It should, but I added env file support

**Review by dekiel — CHANGES_REQUESTED** [review]

**Review by dekiel — DISMISSED** [review]

File: .github/workflows/pull-image-builder-test.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      context: .
+      env-file: ".env"
```
Comment (Sawthis) [inline .github/workflows/pull-image-builder-test.yml:19]:
```suggestion
      env-file: "envs"
```

File: .github/actions/image-builder/action.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  export-tags:
+    description: Export parsed tags as build-args into dockerfile. Each tag will have format TAG_x, where x is the tag name passed along with the tag
+    required: false
+    default: false
+  config:
+    description: Path to the config file
+    required: false
+    default: "./configs/kaniko-build-config.yaml"
+  env-file:
+    description: Path to file with environment variables to be loaded in build
```
Comment (Sawthis) [inline .github/actions/image-builder/action.yml:38]:
```suggestion
    description: Path to file with environment variables to be loaded in the build
```

**Review by Sawthis — CHANGES_REQUESTED** [review]

**Review by Sawthis — APPROVED** [review]

**Pull request merged**